homework-submission/week1/src/Todo.js

```diff
@@ -0,0 +1,44 @@
+import React, {
+    Component
+} from 'react';
+import Main from './Main';
+
+class Todo extends Component {
+
+
+    render() {
+        const Items = [{
+            todoDescription: " Get out of bed",
+            deadline: " Wed Sep 13 2017"
+        },
+        {
+            todoDescription: " Brush teeth",
+            deadline: " Thu Sep 14 2017"
+        },
+        {
+            todoDescription: " Eat breakfast",
+            deadline: " Fri Sep 15 2017"
+        }
+        ];
+
+        return (
+            <div > {
+                Items.map((element, index) => (<
+                    Main key={
+                        index
+                    }
+                    todoDescription={
+                        element.todoDescription
+                    }
+                    deadline={
+                        element.deadline
+                    }
+                />
```

Comment:
This is not the best practice to write a component element, instead write it as normal

```
<Main key={index} todoDescription={element.todoDescription} deadline={element.deadline}/>
```
or

```
<Main
  key={index}
  todoDescription={element.todoDescription}
  deadline={element.deadline}
/>
```